auth/services/oauth/interface.go

```diff
@@ -55,3 +59,17 @@ type Verifier interface {
 	// ClientMetadataURL constructs the URL to the client metadata of the local verifier.
 	ClientMetadataURL(webdid did.DID) (*url.URL, error)
 }
+
+// Holder implements the OpenID4VP Holder role which acts as Authorization server in the OpenID4VP flow.
```

Comment:
OpenID4VCI also defines holder, so naming will clash if this type is only intended to hold OpenID4VP stuff

Reply:
OpenID4VCI oauth2 related stuff should go here as well.